### PR TITLE
Updated Chan1 M3 test to use base lro dem 

### PR DESCRIPTION
**Pull request by krlberry**
## Description
The two test cubes used for the Chandrayaan1 M3 test used an internal-only, 35 GB DEM that is not distributed with ISIS. This caused this test to fail on jenkins, as there is no access to the `/work` area these DEMs were stored in. These cubes were updated to use the base-area lro dem which is already distributed with ISIS. This updated caused the output values for the unitTest calculations to be slightly different. This PR updates the unitTest with these new values. 

## Related Issue
#641 

## Motivation and Context
This issue came up in the context of trying to fix failing jenkins tests. 

## How Has This Been Tested?
Locally, the updated test passes. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/CMakeLists.txt [hunk 1]
```diff
@@ -40,8 +40,8 @@ set(PACKAGE_VERSION    ${VERSION})
 set(PACKAGE_STRING     "${PACKAGE_NAME} ${VERSION}")
 
 # Other release information
-set(VERSION_DATE              "2018-10-26")
-set(RELEASE_STAGE             "beta") # (alpha, beta, stable)
```
Comment (acpaquette) [inline isis/CMakeLists.txt]: Should probably rebase this? I'm guessing it's already in the system but I could be wrong

File: isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth [hunk 2]
```diff
@@ -32,8 +32,8 @@ DeltaLine = 0.000000000
 For center pixel position ...
 Latitude OK
 Longitude OK
-RightAscension = 242.182569911
-Declination = -84.617919659
```
Comment (acpaquette) [inline isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth:36]: So are these values changing due to updated spice data?

File: isis/CMakeLists.txt (hunk 1 above)
Comment (krlberry) [reply isis/CMakeLists.txt]: Oops, yes it is. Thanks!

File: isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth (hunk 2 above)
Comment (krlberry) [reply isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth:36]: I believe so, yes.

File: isis/CMakeLists.txt (hunk 1 above)
Comment (krlberry) [reply isis/CMakeLists.txt]: My attempt at fixing this did not work. Is there any chance you can tell what I did wrong?

**Review by acpaquette — APPROVED** [review]
Previous comments handled. Looks good!

File: isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth (hunk 2 above)
Comment (scsides) [reply isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth:36]: This is not a trivial change, but also not out of the ordinary when updating spice spk and/or ck, and/or dem

**Review by scsides — APPROVED** [review]

**Pull request merged**